meta-resin-common/recipes-support/resin-vars/resin-vars/config-json.path

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Config.json path watch
+Requires=mnt-boot.mount
+After=mnt-boot.mount
+
+[Path]
+PathChanged=/mnt/boot/config.json
+
+[Install]
+WantedBy=mnt-boot.mount
```

Comment:
Any reason to add this dependency to mnt-boot? We can use the default multi-user target because this looks like a dependency loop now.